test/utility/fragments.ts

```diff
@@ -1127,6 +1143,17 @@ export const fundingAccount = gql`
   }
 `;
 
+export const defaultMarketingRegion = gql`
+  fragment defaultMarketingRegion on Location {
```

Comment:
This isn't really the best name. Nothing about the location shape that is defined here is specific to marketing region.
Maybe `locationName`?